create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/utils.py

```diff
@@ -1,15 +1,15 @@
 import argparse
 import os
 import pathlib
-from typing import Any
+from typing import Any, Optional, Dict
 
 import trafaret
 from aiohttp import web
 from trafaret_config import commandline
 
 
 PATH = pathlib.Path(__file__).parent.parent
-settings_file = os.environ.get('SETTINGS_FILE')
+settings_file = os.environ.get('SETTINGS_FILE', '')
```

Comment:
I think `''` is not a good choice for default, because it renders `DEFAULT_CONFIG_PATH` var point to a dir